### DISCUSSION TITLE
how can i use console.log?

### DISCUSSION THREAD

**Discussion by DLX23** [understrap/understrap-child#382] [Q&A] [answered]
hi,

if i use console.log in custom-javascript.js it will be stripped out when i compile.
but i need it for debugging. is there a way to not cut it off?

**Comment by bacoords** [comment, marked as answer]
Yep, answer here in the docs: https://docs.understrap.com/#/issues?id=my-consolelog-output-is-not-showing-up

**Reply by DLX23** [reply]
Oh Sorry! Thank you <3
